database/seeds/RolesTableSeeder.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class RolesTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('roles')->insert([
```

Comment:
dùng factory and facker

Reply:
Vâng. Như trong user thì e có dungf factory and facker nhưng trong role ít bản ghi nên e sử dụng như thế :v